src/main/java/org/opensearch/ad/ratelimit/RateLimitedRequestWorker.java

```diff
@@ -559,7 +559,7 @@ protected void process() {
                 LOG.error(String.format(Locale.ROOT, "Failed to process requests from %s", getWorkerName()), e);
                 if (e != null && e instanceof TimeSeriesException) {
                     TimeSeriesException adExep = (TimeSeriesException) e;
-                    nodeStateManager.setException(adExep.getAnomalyDetectorId(), adExep);
+                    nodeStateManager.setException(adExep.getConfigId(), adExep);
```

Comment:
This is breaking the pattern from getID - are ConfigID and getID related in the request?

Reply:
When it is clear what the ID is, I use getID like detector.getId. Otherwise, I will add what kind of id it is. Here there is some exception and we get the config id.

Reply:
Makes sense